File: test-vercel.js
const test = async () => {
  try {
    const res = await fetch("https://voice-cloning-ai-beige.vercel.app/api/session", { method: "POST" });
    const text = await res.text();
    console.log("Status:", res.status);
    console.log("Body:", text);
  } catch(e) {
    console.error(e);
  }
};
test();

File: test_vercel.py
import urllib.request

req = urllib.request.Request("https://voice-cloning-ai-beige.vercel.app/api/session", method="POST")
try:
    with urllib.request.urlopen(req) as response:
        print("Status:", response.status)
        print("Body:", response.read().decode('utf-8'))
except urllib.error.HTTPError as e:
    print("Status:", e.code)
    print("Body:", e.read().decode('utf-8'))
except Exception as e:
    print("Error:", e)
